tests/test_local_executor.py

```diff
@@ -138,8 +138,21 @@ def test_read_metadata_path(job_definition):
 def test_prepare_success(
     docker_cleanup, job_definition, test_repo, tmp_work_dir, freezer
 ):
-    job_definition.inputs = ["output/input.csv"]
     populate_workspace(job_definition.workspace, "output/input.csv")
+    # this job requires input files from a previous job
```

Comment:
I wonder if we should update populate_workspace to require a job  to files mapping, and do this logic of creating past metadata.json for each job?

I think we'll need it a lot - its already used twice here.